schema/release-schema.json

```
@@ -13,7 +13,7 @@
     },
     "id": {
       "title": "Release ID",
-      "description": "The identifier of the release. The release ID must be unique within the scope of the (contracting or planning) process, for a given version of OCDS. In other words, a publisher may publish datasets for different versions of OCDS, and repeat releases within each dataset. The release ID must not contain the number sign (#). For a compiled release, the `ocid` and the maximum `date` among the individual releases used to create the compiled release, separated by a hyphen: {ocid}-{date}.",
+      "description": "The identifier of the release or a compiled release. For an individual release, the release ID must be unique within the scope of the contracting (or planning) process, for a given version of OCDS. In other words, a publisher may publish datasets for different versions of OCDS, and repeat releases within each dataset. The release ID must not contain the number sign (#). For a compiled release, the release ID should be the `ocid` and the maximum `date` among the individual releases used to create the compiled release, separated by a hyphen: {ocid}-{date}.",
```

The placement of the parentheses is deliberate: https://ocds-standard-development-handbook.readthedocs.io/en/latest/meta/style_guide.html#word-choice

> “(contracting or planning) process”, if the sentence relates to a scope of uniqueness

This was discussed in https://github.com/open-contracting/standard/issues/1409#issuecomment-1129735868 (point 4), as a way to avoid the interpretation that uniqueness bridges both the planning and contracting process relating to the same real-world procedure.

```suggestion
      "description": "The identifier of the individual release or compiled release. For an individual release, the release ID must be unique within the scope of the (contracting or planning) process, for a given version of OCDS. In other words, a publisher may publish datasets for different versions of OCDS, and repeat releases within each dataset. The release ID must not contain the number sign (#). For a compiled release, the release ID should be the `ocid` and the maximum `date` among the individual releases used to create the compiled release, separated by a hyphen: {ocid}-{date}.",
```